packages/server/src/queries/complex/parameter-types.ts

```diff
@@ -7,3 +7,7 @@ export type UserOsmoAddress = z.infer<typeof UserOsmoAddressSchema>;
 export const UserOsmoAddressSchema = z.object({
   userOsmoAddress: z.string().startsWith("osmo").optional(),
 });
+export type ContractOsmoAddress = z.infer<typeof ContractOsmoAddressSchema>;
+export const ContractOsmoAddressSchema = z.object({
+  contractOsmoAddress: z.string().startsWith("osmo").optional(),
+});
```

Comment:
These are the same. It's a nit, but perhaps we can rename and generalize `UserOsmoAddressSchema` > `OsmoAddressSchema` into a single schema.

Reply:
https://github.com/osmosis-labs/osmosis-frontend/pull/3321/commits/0c9ed586bc14c2fbcbeea55426accdc973088b6a